projects/storefrontlib/src/cms-components/product/carousel/product-references/product-references.component.service.ts

```diff
@@ -32,6 +34,34 @@ export class ProductReferencesService {
     );
   }
 
+  getDisplayProductTitles(): Observable<boolean> {
+    return this.displayProductTitles$.pipe(
+      map(data => Boolean(JSON.parse(data.toLowerCase())))
+    );
+  }
+
+  setDisplayProductTitles(): void {
```

Comment:
I'm not a fan of this, by conventions setters will normally take a value as parameter. Please rename to something like `load...` or `fetch...`